removestar/removestar.py

```diff
@@ -67,12 +67,12 @@ def fix_code(code, *, file, max_line_length=100, verbose=False, quiet=False, all
 
         repls[mods[-1]].append(name)
 
-    new_code = replace_imports(code, repls, file=file, verbose=verbose,
-                               quiet=quiet, max_line_length=max_line_length)
+    output = replace_imports(code, repls, file=file, verbose=verbose,
+                               quiet=quiet, max_line_length=max_line_length,**kws_replace_imports)
 
-    return new_code
+    return output
 
-def replace_imports(code, repls, *, max_line_length=100, file=None, verbose=False, quiet=False):
+def replace_imports(code, repls, *, max_line_length=100, file=None, verbose=False, quiet=False,return_replacements=False):
```

Comment:
I had a look again. As I see it, refactoring `replace_imports` for `removestar_nb` might be problematic because it would need duplicating the code. I agree that the keyword arguments that change the return type are not ideal. But I am not sure if there's any alternative here.